docs/azure_arc_jumpstart/azure_arc_servers/day2/arc_automanage/arc_automanage_machine_configuration_custom_linux/_index.md

```diff
@@ -311,34 +334,34 @@ if ($roleDefinitionIds.Count -gt 0)
 
 Check policy compliance by following these steps:
 
-- In the Azure portal, navigate to *Policy* -> *Compliance*
+- In the Azure Portal, navigate to *Policy* -> *Compliance*
 - Set the scope to the resource group your instance of ArcBox is deployed to
 - Filter for *(AzureArcJumpstart) [Linux] Custom configuration*
 
-![Screenshot of Azure portal showing Azure Policy compliance](./14.png)
+![Screenshot of Azure Portal showing Azure Policy compliance](./14.png)
 
 It may take 15-20 minutes for the policy remediation to be completed.
 
 Get a Machine Configuration specific view by following these steps:
 
-- In the Azure portal, navigate to *Azure Arc* -> *Servers*
-- Click on ArcBox-Ubuntu-01 -> Machine Configuration
-- If the status for *ArcBox-Ubuntu-01/AzureArcJumpstart_Linux* is not *Compliant*, wait a few more minutes and click *Refresh*
+- In the Azure Portal, navigate to *Azure Arc* -> *Servers*
+- Click on Arcbox-Ubuntu-01 -> Machine Configuration
```

Comment:
this is not ArcBox

Reply:
The scenario is using ArcBox as a prerequisite:
https://github.com/Azure/arc_jumpstart_docs/blob/automanage_machine_configuration_updates/docs/azure_arc_jumpstart/azure_arc_servers/day2/arc_automanage/arc_automanage_machine_configuration_custom_linux/_index.md?plain=1#L17